src/llmcompressor/modifiers/awq/base.py

```diff
@@ -473,10 +475,15 @@ def _apply_smoothing(self, model: Module) -> None:
                 # [STEP 3]: Compute output of module
                 # could cache from hook, rather than recomputing here
                 fp16_output = self._run_samples(parent_module)
-                fp16_output = fp16_output.clip(
-                    torch.finfo(fp16_output.dtype).min,
-                    torch.finfo(fp16_output.dtype).max,
-                )
+                if fp16_output.shape[0] == 0:
```

Comment:
Slightly more robust
```suggestion
                if fp16_output.numel() == 0:
```

Reply:
updated